rcl/src/rcl/publisher.c

```diff
@@ -203,6 +203,19 @@ rcl_publisher_get_default_options()
   default_options.qos = rmw_qos_profile_default;
   default_options.allocator = rcl_get_default_allocator();
   default_options.rmw_publisher_options = rmw_get_default_publisher_options();
+
+  // Load disable flag to LoanedMessage via environmental variable.
+  bool disable_loaned_message = false;
+  rcl_ret_t ret = rcl_get_disable_loaned_message(&disable_loaned_message);
+  if (ret == RCL_RET_OK) {
+    default_options.disable_loaned_message = disable_loaned_message;
+  } else {
+    RCUTILS_SAFE_FWRITE_TO_STDERR("Failed to get disable_loaned_message: ");
+    RCUTILS_SAFE_FWRITE_TO_STDERR(rcl_get_error_string().str);
```

Comment:
I will use `RCUTILS_SAFE_FWRITE_TO_STDERR_WITH_FORMAT_STRING` instead, but it's OK.